mycodo/controllers/controller_trigger.py

```diff
@@ -226,7 +229,7 @@ def initialize_variables(self):
         elif self.trigger_type == 'trigger_run_pwm_method':
             self.unique_id_1 = self.trigger.unique_id_1
             self.unique_id_2 = self.trigger.unique_id_2
-            self.unique_id_2 = self.trigger.unique_id_3
```

Comment:
Was that typo? Init worked, but loop complained about an unknown id. This change resolved it.